infra/modules/service/main.tf

```
@@ -113,13 +126,20 @@ resource "aws_ecs_task_definition" "app" {
           "awslogs-region"        = data.aws_region.current.name,
           "awslogs-stream-prefix" = local.log_stream_prefix
         }
-      }
-      mountPoints    = []
+      },
+      mountPoints    = [for e in local.ephemeral_write_volumes : e.mount_point]
```

style nit
```suggestion
      mountPoints    = [for volume in local.ephemeral_write_volumes : volume.mount_point]
```

or if we end up renaming local.ephemeral_write_volumes to local.ephemeral_write_volume_configs then do one of `[for config in ...]` or `[for volume_config in ...]`